ServerlessBenchmark/TriggerTests/BaseTriggers/QueueTriggerTest.cs

```diff
@@ -174,40 +174,37 @@ await CloudPlatformController.EnqueueMessagesAsync(new CloudPlatformRequest()
         }
 
         private async Task<bool> VerifyQueueMessagesExistInTargetQueue(int expected)
```

Comment:
You are changing the semantics of the method. Now after it finishes items are still on the target queue. 
How about recreating the whole queue to make sure it's empty? There might be some time you need to wait between deleting and creating new one.